src/services/storage/index.ts

```diff
@@ -143,18 +145,39 @@ export class StorageService {
          *
          * @throws {Error} When an exception occurs.
          */
-        read: (id: number): Promise<Event> => {
-            return this.query.get<Event>('SELECT * FROM events WHERE id=?', id);
+        read: async (id: number): Promise<Event> => {
+            const data = await this.query.get<Event>('SELECT * FROM events WHERE id=?', id);
+            data.data = JSON.parse(String(data.data));
+            return data;
+        },
+        /**
+         * Display the transactions of a specified address.
+         *
+         * @param address The address of the account which transactions you want to see.
+         *
+         * @throws {Error} When an exception occurs.
+         */
+        transactions: async (address: string): Promise<Event[]> => {
```

Comment:
Wat is de reden dat je deze bij events hebt gezet en niet bijvoorbeeld een speciale `transactions` object hebt aangemaakt? En willen we als deze methoden aanroepen dan ook niet juist `Transaction` objecten terugkrijgen?